components/pager/overview.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Pager
+page_title: Pager for Blazor Overview
+description: Add a page navigation to Blazor application
+slug: components-pager-overview
+tags: telerik,blazor,pager,paging
+published: True
+position: 20
+---
+
+# Pager Overview
+
+The `Pager` component will enable you to add page navigation in your Blazor application. It follows the design patter of our components but as standalone one can be used outside of Grid.
+
+To use Telerik Pager component for Blazor:
+1. Add the `TelerikPager` tag
+1. (optional) Populate it's `Total` parameter
+1. (optional) Set it's `ButtonCount` parameter
+1. (optional) Set the `PageChanged` parameter
+1. (optional) Set `Page` or `@bind-Page` (one and two-way data binding)
+1. (optional) Set a `PageSize`
+
+>caption Basic setup of Pager component in your application
+
+````CSHTML
+@*Basic Pager configuration.*@
+
+<TelerikPager Total="TotalItems"
+              ButtonCount="ButtonCount"
+              PageSize="ItemsOnPage"
+              @bind-Page="CurrentPage">
+
+</TelerikPager>
+
+@code {
+    public int TotalItems { get; set; } = 80;
+    public int ButtonCount { get; set; } = 4;
+    public int ItemsOnPage { get; set; } = 10;
+    public int CurrentPage { get; set; } = 2;
+}
+````
+
+>caption The result from the code snippet above
+
+![basic configuration of the pager](images/pager-basic-configuration-screenshot.jpg)
+
+## Features
+* `Class` - The CSS class that will be rendered on the main wrapping element of the Pager.
+* `Total` - **int** - Represents the total count of items in the pager.
+* `ButtonCount` - **int** - The number of pages to be visible. To take effect the `ButtonCount` must be **less** than the pages count (ButtonCount < Total / number of items on the page)
+* `Page` and `@bind-Page` - **int** - Represents the current page of the pager. Those parameters are respectively for one and two-way data binding. If no `Page` or `@bind-Page` are provided they will default to the first page (1).
+* `PageChanged` - Fires when a new page is selected (used in one-way data binding).
+* `PageSize` - **int** - The number of items to be presented on a pages.
+
+## Examples
```

Comment:
The first example should be something similar, but also include a `.Skip().Take()` call from an AllData collection into an CurrentPageData collection that you can use for the loop.

A second example here can call an `async` method that will return data based on the given PageSize and Page parameters so you can simulate load-on-demand from a service with it. Title the example accordingly.